test/harvestFi-strat-migration.js

```diff
@@ -0,0 +1,88 @@
+const hre = require('hardhat')
+const { describe, it } = require('mocha')
+const { ethers } = require('hardhat')
+const { expect } = require('chai')
+
+const VAULT = '0x2dCdCA085af2E258654e47204e483127E0D8b277'
```

Comment:
We should test one of the WBTC or YFI vault since these are the one's we're more likely to migrate from